hookman/plugin_config.py

```diff
@@ -1,42 +1,84 @@
 import ctypes
 import sys
 from pathlib import Path
+from typing import List
 from zipfile import ZipFile
 
 import attr
 from attr import attrib
 
 from hookman.exceptions import PluginNotFoundError
+from hookman.hookman_utils import load_shared_lib
 
+@attr.s(frozen=True, auto_attribs=True)
+class ConflictStatus(object):
+    """
+    Class that hold information related to conflict between plugins.
+    """
+    plugins : List[str]
+    hook : str
 
 @attr.s(frozen=True)
 class PluginInfo(object):
     """
     Class that holds all information related to the plugin with some auxiliary methods
     """
     location = attrib(type=Path)
+    hooks_available = attrib(type=dict)
```

Comment:
Where is this set? I couldn't find it in the code.

Reply:
It's created on the method `plugins_available` as can be checked below:

```Python
    def plugins_available(self) -> Optional[List[PluginInfo]]:
        """
        Return a list with all plugins that are available on the plugins_dirs.
        The list contains a PluginInfo object that contains information about the plugin
        (configuration available at the YAML file and computed values )
        """
        plugin_config_files = hookman_utils.find_config_files(self.plugins_dirs)
        return [PluginInfo(plugin_file, self.hooks_available) for plugin_file in plugin_config_files]
```